k2/python/k2/fsa_algo.py

```diff
@@ -60,7 +61,10 @@ def top_sort(fsa: Fsa) -> Fsa:
 
 
 def intersect(a_fsa: Fsa, b_fsa: Fsa,
-              treat_epsilons_specially: bool = True) -> Fsa:
+              treat_epsilons_specially: bool = True,
+              seqframe_idx_name: Optional[str] = None,
```

Comment:
Could you add some doc for these arguments?

Reply:
Sorry, remove this, it should only have been in intersect_dense.